project-management/src/main/java/com/smartagilify/projectmanagement/services/projectmember/ProjectMemberServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.smartagilify.projectmanagement.services.projectmember;
+
+import com.smartagilify.core.enumerations.EN_STATE;
+import com.smartagilify.core.services.BaseService;
+import com.smartagilify.projectmanagement.exceptions.ApplicationException;
+import com.smartagilify.projectmanagement.mappers.projectmember.ProjectMemberMapper;
+import com.smartagilify.projectmanagement.model.dtos.ProjectMemberDTO;
+import com.smartagilify.projectmanagement.model.entities.Project;
+import com.smartagilify.projectmanagement.model.entities.ProjectMember;
+import com.smartagilify.projectmanagement.repositories.project.ProjectRepository;
+import com.smartagilify.projectmanagement.repositories.projectmember.ProjectMemberRepository;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+
+@Service
+public class ProjectMemberServiceImpl extends BaseService<ProjectMember, ProjectMemberMapper, ProjectMemberDTO>
+        implements ProjectMemberService {
+    private final ProjectMemberRepository projectMemberRepository;
```

Comment:
isn't it `jpaRepository`? i think you can cast `jpaRepository `to use it properly